Earthfile

```diff
@@ -0,0 +1,10 @@
+VERSION 0.6
+
+build:
+    FROM bufbuild/buf
+    COPY --dir api/translate .
+    WORKDIR translate
+    RUN buf mod update
+    RUN buf build
+    RUN buf generate
```

Comment:
Can you please mount the buf dependencies directory with type=cache? To avoid downloading all dependencies every time we make changes to proto files.

Reply:
Want to discuss about this

Reply:
https://docs.buf.build/ci-cd/setup#ci-caching

Reply:
Should be okay now